include/sapi/marshal.h

```diff
@@ -834,6 +862,20 @@ TPMS_TAGGED_PCR_SELECT_Unmarshal(
     TPMS_TAGGED_PCR_SELECT *dest);
 
 TSS2_RC
+TPMS_QUOTE_INFO_Marshal (
```

Comment:
Same as last comment but should be in b41f17fd471a836ce423af1a9ef1f35780fee994 instead?